draft-ietf-jose-hpke-encrypt.md

```diff
@@ -433,16 +444,15 @@ In Key Agreement with Key Wrapping mode, CEK has to be randomly generated and it
 
 ## HPKE authentication
 
-Authenticated HPKE modes MUST NOT be used with Key Encryption, as this is trivially insecure.
+Authenticated HPKE modes MUST NOT be used for Key Encryption, as the message is not authenticated. Any recipient could act as a man-in-the-middle (MitM) and modify the message.
 
 ## Key Management
 
-A single key SHOULD NOT be used as both authentication key and recipient key, as this has not been proven to be safe.
+A single key MUST NOT be used in both sender and recipient roles. Avoiding the use of the same key for both sender and recipient roles ensures clear cryptographic boundaries and minimizes unintended interactions.
```

Comment:
Why? isn't it viable to encrypt messages to yourself?

Reply:
"encrypt to yourself" is usually done using symmetric algorithms, and this is the only known way of doing it post-quantum.

However, the algorithms available for this in JOSE are not great. Only the AES-CBC-HMAC is suitable for Direct Encryption (it is non-trivial to use AES-GCM in Direct Encryption without NDA attack).